### PR TITLE
upgraded sample project to .NET 8

### PR DESCRIPTION
**Pull request by arya2004**
### Brief Summary of Changes

- Updated the project to be compatible with **.NET 8**, resolving outdated dependencies .
- Updated the documentation to be suited for .NET 8.

#### What does this PR address?
- [x] GitHub issue #368
- [ ] Refactoring
- [ ] New feature
- [ ] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [ ] Yes
- [x] No

#### Reviewer, please note:

- This PR focuses solely on updating the project to .NET 8. No new features are included yet.
- Additional features (like the image gallery and Cloudinary integration) will be added in subsequently.

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

**Comment by cld-sec** [comment]
### 🎉 Snyk hasn't found any issues so far.

✅ **security/snyk** check is completed. **No issues were found**. [(View Details)](https://app.snyk.io/org/cloudinaryltd/pr-checks/52d81790-31e7-4e24-9cc9-71765b845960?refs=ee18c1d684330d72b5a3c1058a3c2db63eedb673&source=prChecksComment)

✅ **license/snyk** check is completed. **No issues were found**. [(View Details)](https://app.snyk.io/org/cloudinaryltd/pr-checks/52d81790-31e7-4e24-9cc9-71765b845960/license?refs=ee18c1d684330d72b5a3c1058a3c2db63eedb673&source=prChecksComment)

**Comment by arya2004** [comment]
I have  updated the project to **.NET 8**, addressing the dependency update requirement. Below are the updates and next steps for my planned contributions:

- [x] **Updated to .NET 8**: 

- [ ] **Basic Gallery with Upload, View, and Editing via Cloudinary**:
  - I am planning to implement a basic image gallery where users can:
    - Upload images to Cloudinary.
    - View images in a gallery interface.
    - Edit images (resize, crop, etc.) using Cloudinary's transformation features.
  - The UI will follow the **Blazor Quick Start project** design.


- [ ] **Advanced Use Cases**:
  - I’ll add advanced examples showcasing:
    - Cloudinary's image optimization and transformations.
    - Additional media management features (videos, etc.), if relevant.

- [ ] **Improved Documentation**


Let me know if this plan aligns with the direction of the project. Once approved, I’ll proceed with implementing these features.

**Comment by const-cloudinary** [comment]
@arya2004 The plan sounds great! Feel free to proceed!

File: samples/PhotoAlbum/Pages/Upload.cshtml.cs [hunk 1]
```diff
@@ -4,13 +4,16 @@
...
 using CloudinaryDotNet.Actions;
+using Markdig;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using PhotoAlbum.Data;
 using UploadResult = PhotoAlbum.Data.UploadResult;
+using Markdig;
```
Comment (const-cloudinary) [inline samples/PhotoAlbum/Pages/Upload.cshtml.cs:15]:
```suggestion
```

File: samples/PhotoAlbum/appsettings.Development.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
     }
   },
   "AccountSettings": {
-    "CloudName": "",
-    "ApiKey": "",
-    "ApiSecret": "",
+    "CloudName": "dtxqasgig",
```
Comment (const-cloudinary) [inline samples/PhotoAlbum/appsettings.Development.json:10]:
```suggestion
    "CloudName": "",
```

File: samples/PhotoAlbum/appsettings.Development.json [hunk 3]
```diff
@@ -7,9 +7,9 @@
...
+    "ApiKey": "533752815973466",
```
Comment (const-cloudinary) [inline samples/PhotoAlbum/appsettings.Development.json:11]:
```suggestion
    "ApiKey": "",
```

File: samples/PhotoAlbum/appsettings.Development.json [hunk 4]
```diff
@@ -7,9 +7,9 @@
...
+    "ApiSecret": "LkhGjqjO8zQkjbzc2hL1jFDc628",
```
Comment (const-cloudinary) [inline samples/PhotoAlbum/appsettings.Development.json:12]:
```suggestion
    "ApiSecret": "",
```

**Review by const-cloudinary — CHANGES_REQUESTED** [review]
@arya2004 The project looks awesome!

There are some broken links on top right corner, not critical.

Please remove your credentials from:
https://github.com/cloudinary/CloudinaryDotNet/pull/371/files#diff-9d2b1915978abac450213acc4fc09a2b268e3ed4db2bff3126f89a3507df2c4a

And I strongly suggest you to regenerate your access key in admin console, since this one is obviously leaked.

Feel free to proceed with your plan (like adding a better image gallery, or maybe adding support for deleting files ( I uploaded some of my images to your cloud :) )

**Comment by arya2004** [comment]
apologies for the mistake ;(

will prioritize minor refactoring....

**Comment by const-cloudinary** [comment]
@arya2004 it looks great! Thank you very much for contribution!

**Comment by colbyfayock** [comment]
@arya2004 thanks for your contribution. this PR is eligible for free Cloudinary Hackatoberfest swag. Please send me an email at [hacktoberfest@cloudinary.com](mailto:hacktoberfest@cloudinary.com) with your name, GitHub username, and a link to the PR where I'll send further instructions.

Happy Hacktoberfest!

**Comment by arya2004** [comment]
@colbyfayock I've sent the email with my details. Looking forward to hearing back!

**Pull request merged**